source/index.md

```diff
@@ -4,27 +4,33 @@ description: 'Meteor Tuts. Learning Meteor From NOOB to PRO'
 disqusPage: 'Welcome'
 ---
 
-<h1>Hello!</h1>
+<h1>Hello stranger!</h1>
```

Comment:
stranger?
Let's pick something more positive: Ninja Coder, any other idea... something to give them a good vibe.